doc/api-extensions.md

```diff
@@ -2501,3 +2501,9 @@ Expands APIs under `/1.0/auth` to include:
   The caller must provide a base64 encoded x509 certificate in the `certificate` field of the request body.
   Fine-grained TLS identities may update their own certificate.
   To update the certificate of another identity, the caller must have `can_edit` on the identity.
+
+## `list_ovn_uplink_allocations`
```

Comment:
Should be called something like `network_allocations_ovn_uplink` so it aligns with "thing_feature" style and with the earlier "network_allocations" extension.